### PR TITLE
Replace `_.zipObject` with vanilla JS

### PR DESCRIPTION
**Pull request by XhmikosR**
TODO:

- [x] decide with which solution to go (1st or 2nd patch); I haven't done any benchmarks, so any help to decide is welcome
- [x] add more tests (help welcome!)

Refs #604

### CODE REVIEW

File: test/utils.test.js [hunk 1]
```diff
@@ -196,4 +202,14 @@ describe('utils', () => {
             expect(isPlainObject(Symbol('test'))).toBe(false);
         });
     });
+
+    describe('zipObject', () => {
+        it('should return the zipped object', () => {
+            expect(zipObject(['a', 'b'], [1, 2])).toStrictEqual({ a: 1, b: 2 });
```
Comment (Kreeg) [inline test/utils.test.js:208]: Need more tests to cover possible variants, like 
+`(['a', 'b', 'c'], [1, 2])`
+`(['a'], [1, 2])`

File: lib/svg-sprite/utils/index.js [hunk 2]
```diff
@@ -41,9 +41,19 @@ function isString(value) {
     return Object.prototype.toString.call(value) === '[object String]';
 }
 
+/**
+ * @param {Array} array1    First array
+ * @param {Array} array2    Second array
+ * @returns {object}        The zipped Object
+ */
+function zipObject(array1, array2) {
```
Comment (Kreeg) [inline lib/svg-sprite/utils/index.js:49]: I'm thinking this arguments should be checked to be an array. Instead throw error.

File: test/utils.test.js (hunk 1 above)
Comment (XhmikosR) [reply test/utils.test.js:208]: Feel free to add them but I'm not sure if they'll work. What did the lodash zipObject return in these cases?

File: lib/svg-sprite/utils/index.js (hunk 2 above)
Comment (XhmikosR) [reply lib/svg-sprite/utils/index.js:49]: I thought about this too, but didn't add this yet. Feel free to push :)

**Comment by Kreeg** [comment]
@XhmikosR I vote for second patch. I think It's just clearer.

File: test/utils.test.js (hunk 1 above)
Comment (XhmikosR) [reply test/utils.test.js:205]: @Kreeg did you confirm what lodash is returning with unbalanced arrays?

Comment (XhmikosR) [reply test/utils.test.js:205]: So, the current behavior is the same, right?

Comment (Kreeg) [reply test/utils.test.js:205]: @XhmikosR yes

**Pull request merged**